modules/simphera_aws_instance/variables.tf

```diff
@@ -85,13 +85,13 @@ variable "postgresqlMaxStorageKeycloak" {
 variable "db_instance_type_keycloak" {
   type        = string
   description = "PostgreSQL database instance type for Keycloak data"
-  default     = "db.t3.large"
+  default     = "db.t4g.large"
```

Comment:
Why db.t4g.large?

Reply:
`$0.0768 for t4g.large` vs `$0.096 for t3.large`,  and "[Amazon EC2 T4g instances](https://aws.amazon.com/ec2/instance-types/t4/) ... deliver up to 40% better price performance over T3 instances for a broad set of burstable general purpose workloads."